src/adapter/mod.rs

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+mod philips_hue;
+
+use controller::Controller;
+use service::ServiceAdapter;
+
+pub struct AdapterManager<T> {
+    controller: T
+}
+
+impl<T: Controller> AdapterManager<T> {
+    pub fn new(controller: T) -> Self {
+        debug!("Creating Adapter Manager");
+        AdapterManager { controller: controller }
+    }
+
+	pub fn start(&self) {
```

Comment:
Done.

Seems it was automatically fixed by my editor in one of my other commits.